Components/Input/input.html

```diff
@@ -0,0 +1,120 @@
+            <p class="container_header_content">Input Container is used to input variables. </p>
+            <hr>
+            <h2 class="container_subheader">Textbox Input</h2>
+            <p class="container_subheader_content">TextBox Input is the easiest way to input any variable in the UI.
+                type='text' defines a single-line input text field whereas type='number' defines number only text field.
+            </p>
+
+            <div class="input_textbox">
+                <div class="input_textbox_textonly">
+                    <input class="input_box" type="text" placeholder="Enter text..">
+                </div>
+                <div class="input_textbox_numberonly">
+                    <input class="input_box" type="number" placeholder="Enter number..">
+                </div>
+            </div>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=night-owl&wt=sharp&l=css&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input_textbox_textonly%2522%253E%250A%2520%2520%2520%253Cinput%2520class%253D%2522input_box%2522%2520type%253D%2522text%2522%2520placeholder%253D%2522Enter%2520text..%2522%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522input_textbox_numberonly%2522%253E%250A%2520%2520%2520%253Cinput%2520class%253D%2522input_box%2522%2520type%253D%2522number%2522%2520placeholder%253D%2522Enter%2520number..%2522%253E%250A%253C%252Fdiv%253E%2520"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <hr>
+            <h2 class="container_subheader">Textbox with Validation and Error Styles</h2>
+            <p class="container_subheader_content">TextBox with validation and error styles is below in which in input tag we have provided required and for error we have made another div tag with class necessary_requirements.
+            </p>
+            <form action="#">
+                <div class="input_textbox">
+                    <input class="input_box" type="text" required placeholder=" Enter Name .. " />
+                </div>
+                <div class="input_textbox">
+                    <input class="input_box" type="email" required placeholder="Enter emailid..">
+                    <div class="necessary_requirements">It must be a valid email address.</div>
+                </div>
+                <div class="input_textbox">
+                    <input class="input_box" type="password" required placeholder=" Enter password.. " />
+                    <div class="necessary_requirements">Your password must be at least 6 characters as well as contain
+                        at least one uppercase, one lowercase, and one number.</div>
+                </div>
+                <div class="input_textbox">
+                    <input class="input_box" type="submit" value="Sign Up">
```

Comment:
IMO, `button` is more semantically correct here

Reply:
replaced `input` with `submit`